style.css

```diff
@@ -584,4 +590,41 @@ i {
     overflow-x: auto;
     display: inline-block;
   }
+
+  #contact {
+    white-space: nowrap;
+    overflow-x: auto;
+    display: inline-block;
+    padding-left: 15px !important;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #contact {
+    display: none;
+  }
+
+  .footer-2 {
+    flex-direction: row;
+    justify-content: space-around;
+    padding: 10px 11.25rem 0;
+  }
+
+  .container-5 {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  .title-4 {
```

Comment:
Please ensure that your media queries for the Desktop breakpoint in the CSS file. (the value for `min-width` property should be in range of `992px` - `1100px`)